src/components/shared/Entity/Actions/useSave.ts

```diff
@@ -263,7 +260,7 @@ function useSave(
             const response = await createPublication(
                 draftId,
                 dryRun ?? false,
-                entityDescription,
+                undefined,
```

Comment:
No longer going to pass details. Want to default this to something - but we can do that later.